smartsim/_core/_install/builder.py

```diff
@@ -233,6 +237,43 @@ def copy_tf_cmake(self):
                 self.bin_path / "modules/FindTensorFlow.cmake", tf_cmake, set_exe=False
             )
 
+    def symlink_libtf(self, device):
+        """Add symbolic link to available libtensorflow in RedisAI deps.
+
+        :param device: cpu or gpu
+        :type device: str
+        """
+        rai_libtf_path = (
+            self.rai_build_path
+            / "deps"
+            / f"{platform.system().lower()}-{platform.machine()}-{device}"
+            / "libtensorflow"
+        )
+        rai_libtf_path.resolve()
+        if rai_libtf_path.is_dir():
+            shutil.rmtree(rai_libtf_path)
+
+        os.makedirs(rai_libtf_path)
+        libtf_path = Path(self.libtf_dir).resolve()
+        os.symlink(libtf_path / "include", rai_libtf_path / "include")
+        # RedisAI expects to find a lib directory, which is only
+        # available in some distributions.
+        rai_libtf_lib_dir = rai_libtf_path / "lib"
+        os.makedirs(rai_libtf_lib_dir)
```

Comment:
May want to include some execption handling here for errors that crop up in directory naming and what not.